extensions/azdata/src/extension.ts

```diff
@@ -85,7 +92,8 @@ function throwIfNoAzdata(): void {
 
 async function checkForAzdata(): Promise<IAzdataTool | undefined> {
 	try {
-		return await findAzdata();
+		const azdata = await findAzdata(); // find currently installed Azdata
+		vscode.window.showInformationMessage(loc.foundExistingAzdata(azdata.path, azdata.cachedVersion.raw));
```

Comment:
Just log this to output channel

Reply:
I think this was addressed in other PR but will do it here also.

Reply:
removed as findAzdata already logs to output channel